packages/components/entity-list/src/EntityListItem/EntityListItem.tsx

```diff
@@ -186,107 +188,109 @@ export const EntityListItem = ({
           </Skeleton.Container>
         </article>
       ) : (
```

Comment:
This looks like a much bigger change than it is.  All I'm doing is adding a `<>` and moving the `<Element ...` and  `{actions && (...` elements alongside each other.